ttnn/cpp/ttnn/operations/normalization/softmax/device/multi_core/softmax_op_multi_core.cpp

```diff
@@ -347,7 +347,7 @@ operation::ProgramWithCallbacks scale_mask_softmax_multi_core(
         uint32_t NCHt = NC*Ht;
         uint32_t num_tile_rows = NC * Ht;
         auto all_device_cores = CoreRange({0, 0}, {grid_size.x - 1, grid_size.y - 1});
-        auto [num_cores, all_cores, core_group_1, core_group_2, num_tile_rows_per_core_group_1, num_tile_rows_per_core_group_2] = split_work_to_cores(grid_size, num_tile_rows, true);
+        auto [num_cores, all_cores, core_group_1, core_group_2, num_tile_rows_per_core_group_1, num_tile_rows_per_core_group_2] = ttnn::operations::core::work_split::split_work_to_cores(grid_size, num_tile_rows, true);
 
```

Comment:
We (@arakhmati  and I) tried to keep the python api to mirror the C++ api.   So if we have ttnn.split_work_to_cores we want to have ttnn::split_work_to_cores.   It's ok to have split_work_to_cores in the namespace ttnn::operations::core::work_split but we can allow it to exist in just ttnn namespace as well.

Reply:
Will Update